src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -247,9 +247,30 @@ <h3>Headers</h3>
                     <!-- /ko -->
                     <!-- ko if: !header.options || header.options.length === 0 -->
                     <div class="input-group">
+                        <!-- ko if: !header.secret -->
                         <input type="text" autocomplete="off" class="form-control form-control-sm" placeholder="value"
                             spellcheck="false" aria-label="Header value"
                             data-bind="event: { keyup: $component.updateRequestSummary }, textInput: header.value">
+                        <!-- /ko -->
+                        <!-- ko if: header.secret -->
+                        <div style="width: 100%;">
```

Comment:
`<div style="width: 100%;">` - we should try to avoid placing styles into the markup and rely more on CSS, where we define style for every component. That allows us to change website/component appearance from one place. Otherwise, it's pretty hard to debug styling issues, especially on a larger scale.